openage/convert/driver.py

```diff
@@ -51,6 +53,21 @@ def get_string_resources(srcdir):
                 # that's fine, there are no language files for every language.
                 pass
 
+        # HD Edition 3.x and below store language .txt files in the Bin/ folder.
+        for lang in srcdir["bin"].list():
```

Comment:
Likely :) And the earlier loop only works for 4.x. I could move both loops into an `if/elif` instead.